README.md

```diff
@@ -36,7 +36,7 @@ Last stable versions of packages that have been provided for usage with Azure an
 
 ### Management: New Releases
 A new set of management libraries that follow the [Azure SDK Design Guidelines for Python](https://azure.github.io/azure-sdk/python/guidelines/) are now available. These new libraries provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more. 
-Documentation and code samples for these new libraries can be found [here](http://aka.ms/azsdk/python/mgmt). In addition, a migration guide that shows how to transition from older versions of libraries is located [here](https://azure.github.io/azure-sdk-for-python/mgmt_preview_quickstart.html#migration-guide). 
+Documentation and code samples for these new libraries can be found [here](http://aka.ms/azsdk/python/mgmt). In addition, a migration guide that shows how to transition from older versions of libraries is located [here](https://github.com/Azure/azure-sdk-for-python/blob/master/doc/sphinx/mgmt_quickstart.rst#migration-guide). 
```

Comment:
Were these changes intentional? If so, it would probably be better with a separate PR since I don't believe they are related to the KeyVault query backup/restore status methods, right?

Reply:
This was definitely unintentional -- I had to rebase to pull some changes in order to get tests to work, and those pulled commits ended up being tacked onto my most recent one. Sorting this out this morning!